examples/examples_test.go

```diff
@@ -74,7 +74,8 @@ func TestExamples(t *testing.T) {
 	shortTests := []integration.ProgramTestOptions{
 		jsBase.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "minimal")}),
 		jsBase.With(integration.ProgramTestOptions{
-			Dir: path.Join(cwd, "serverless"),
+			Dir:           path.Join(cwd, "serverless"),
+			RunUpdateTest: true,
```

Comment:
Similar feedback here, would be nice to move RunUpdateTest to the `jsBase` thing.

Reply:
Same response as with AWS - will test and come back to make sure we do this if it works